src/main/java/org/spongepowered/server/mixin/entity/player/MixinEntityPlayer.java

```diff
@@ -110,6 +119,16 @@ public void onWriteEntityToNBT(NBTTagCompound tagCompound, CallbackInfo ci) {
         tagCompound.setTag("Spawns", spawnlist);
     }
 
+    @Inject(method = "interactWith", at = @At(value = "INVOKE_ASSIGN", target = "net/minecraft/entity/player/EntityPlayer"
```

Comment:
`INVOKE` would be enough here as we don't need the item stack for the event (right now this is injected _after_ the item stack declaration, `INVOKE` will inject it _before_ the declaration).